### PR TITLE
Update dependencies, format code.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

**Comment by mrstegeman** [comment]
@andrenatal ping

File: server.js [hunk 1]
```diff
@@ -372,102 +413,114 @@ app.post('*', function (req, res, next) {
...
+            });
+            return next(s3_error);
+          }
+
+          mozlog.info('request.s3.json.finish', {
             request_id: res.locals.request_id,
             key: key_base + '/transcript.json',
-            status: s3_error.statusCode,
-            time: Date.now() - s3_request_start
+            status: 200,
```
Comment (andrenatal) [inline server.js]: are we sure this always return 200?

Comment (mrstegeman) [reply server.js]: The diff is just weird here. This maps to the old line 449, which is identical.

**Comment by mrstegeman** [comment]
The status code didn't change, the diff is just a bit confusing.

**Comment by andrenatal** [comment]
@mrstegeman This PR broke the service. See: https://github.com/mozilla/speech-proxy/pull/82#issue-243195395

Any idea? Will revert it until we figure what's wrong.

**Comment by mrstegeman** [comment]
@andrenatal It looks like express may have broken the `res.locals` object or something. The code changes were insignificant, so the dependency upgrades must have broken.

I've restored the formatting changes here: https://github.com/mozilla/speech-proxy/pull/84

We can just leave out the dependency upgrades for the time being.

**Pull request merged**